src/lib/marketplace/validation.ts

```diff
@@ -51,18 +54,27 @@ export function removeApiBorderDuplicates(licenses: readonly License[]) {
           data: a,
           diff,
         };
-        throw new AttachableError('Duplicate deals found at API border', JSON.stringify(json, null, 2));
+         // If duplicate deals exist delete them and push them to duplicateLicenses Array
+        dups.forEach((dup) => duplicateLicenses.push(dup));
+        strippedDups.splice(0);
+        console.printWarning("Duplicate Deals", "Duplicate deals found at API border", JSON.stringify(json, null, 2));
+        void slack?.notifyWarning("Duplicate deals found at API border", JSON.stringify(json, null, 2));
+        // throw new AttachableError('Duplicate deals found at API border', JSON.stringify(json, null, 2));
       }
     }
-
     // Keep the first one with attributions
-    dups.sort((a, b) => a.data.evaluationOpportunitySize ? -1 : 1);
-    assert.ok(dups[0].data.evaluationOpportunitySize);
-    dups.length = 1;
+    if (strippedDups.length) {
+      dups.sort((a, b) => a.data.evaluationOpportunitySize ? -1 : 1);
+      assert.ok(dups[0].data.evaluationOpportunitySize);
+      dups.length = 1;
+    }
   }
 
   const fixed = Object.values(groups);
-  assert.ok(fixed.every(ls => ls.length === 1));
+  // Check if duplicate licenses exist and if they do generate array without them
+  const fixedDups = fixed.filter(license => !duplicateLicenses.includes(license[0]));
+  if (fixedDups) assert.ok(fixedDups.every(ls => ls.length === 1));
+  if (!fixedDups) assert.ok(fixed.every(ls => ls.length === 1));
```

Comment:
`fixedDups` -> `fixedDups.length`
`!fixedDups` -> `!fixedDups.length`

https://medium.com/coding-in-simple-english/what-are-truthy-values-in-javascript-e037bdfa76f8